crates/ink/codegen/src/generator/mod.rs

```diff
@@ -70,3 +73,32 @@ pub use self::{
     storage_item::StorageItem,
     trait_def::TraitDefinition,
 };
+
+/// Check if the the conditional compilation expression
+/// for any `cfg` attributes is satisfied.
```

Comment:
As I read the implementation, it is actually "check if any cfg predicate is not satisfied"